src/main/java/sx/blah/discord/handle/impl/obj/Channel.java

```diff
@@ -516,7 +516,7 @@ public void unpin(IMessage message) throws RateLimitException, DiscordException,
 
 	@Override
 	public boolean isDeleted() {
-		return getClient().getChannelByID(id) != this;
+		return getGuild().getChannelByID(id) != this;
```

Comment:
You should add an override to this for VoiceChannel as VoiceChannels are stored in a separate list than the text channels.